internal/pkg/cli/storage_init.go

```diff
@@ -636,6 +716,10 @@ func (o *initStorageOpts) Execute() error {
 }
 
 func (o *initStorageOpts) readWorkloadType() error {
+	if !o.workloadExists {
+		o.workloadType = workloadTypeNonLocal
```

Comment:
what's the point of type non-local? is it going to be used in a follow-up PR?

Reply:
Not really!

I thought of `o.workloadType == ""` to mean that "the workload type is not read yet", which is different than that "the workload type does not need to be read". So I assigned it a value so that it is not `""`, which could potentially communicate the wrong message.